src/sagemaker/workflow/model_step.py

```diff
@@ -50,10 +50,17 @@ def __init__(
         Args:
             name (str): The name of the `ModelStep`. A name is required and must be
                 unique within a pipeline.
-            step_args (_ModelStepArguments): The arguments for the `ModelStep` definition.
-            depends_on (List[Union[str, Step, StepCollection]]): A list of `Step`/`StepCollection`
-                names or `Step` instances or `StepCollection` instances that the first step,
-                in this `ModelStep` collection, depends on.
+            step_args (_ModelStepArguments): The arguments for the `ModelStep` definition,
+                generated by invoking the :func:`~sagemaker.model.Model.register` or
+                :func:`~sagemaker.model.Model.create`
+                under the :class:`~sagemaker.workflow.pipeline_context.PipelineSession`. Example::
+
+                     model = Model(session=PipelineSession())
+                     model_step = ModelStep(step_args=model.register())
```

Comment:
model = Model(sagemaker_session=PipelineSession()) ?

Reply:
updated it.